cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -79,22 +170,22 @@ private JsonObject generateBrokerCapacity(int brokerId, JsonObject diskCapacity,
      * @param idx Index of the broker
      * @return Disk capacity configuration value as a JsonObject for broker idx
      */
-    private JsonObject generateJbodDiskCapacity(Storage storage, int idx) {
-        JsonObject json = new JsonObject();
+    private static JsonObject generateJbodDiskCapacity(Storage storage, int idx) {
```

Comment:
In Kafka we typically talk about broker id rather than broker index. Should we rename the argument to `brokerId`?